### PR TITLE
Add `Wiki Contribution Guide` page

### PR DESCRIPTION
**Pull request by BlueChinchompa**
its not finished but its a start

### CODE REVIEW

File: wiki/Wiki_Contribution_Guide/en.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Titanic! Wiki Contribution Guide
+
+Thanks for your interest in helping improve the Titanic! wiki. We really appreciate it.
```
Comment (iSlodinxOsu) [inline wiki/Wiki_Contribution_Guide/en.md:3]:
```suggestion
Thanks for your interest in helping improve the Titanic! wiki. We really appreciate it.
```

File: wiki/Wiki_Contribution_Guide/en.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Titanic! Wiki Contribution Guide
+
+Thanks for your interest in helping improve the Titanic! wiki. We really appreciate it. 
+This is a step by step guide on how you can help make changes to the wiki.
```
Comment (iSlodinxOsu) [inline wiki/Wiki_Contribution_Guide/en.md:4]:
```suggestion
This is a step-by-step guide on how you can help make changes to the wiki.
```

File: wiki/Wiki_Contribution_Guide/en.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+
+To begin with you will need to have a basic understanding of GitHub or [git](https://git-scm.com/). It's okay if you get confused you can always ask in the [Titanic! discord](https://discord.gg/qupv72e7YH) in the `#wiki` channel.
```
Comment (iSlodinxOsu) [inline wiki/Wiki_Contribution_Guide/en.md:6]:
```suggestion
To begin with you will need to have a basic knowledge of GitHub or [git](https://git-scm.com/). It's okay if you get confused, you can always ask in the [Titanic! Discord server](https://discord.gg/qupv72e7YH) in the `#wiki` channel.
```

**Review by iSlodinxOsu — CHANGES_REQUESTED** [review]

**Comment by Lekuruu** [comment]
Let's convert this into a draft for now.

**Pull request closed without merging**